src/main/java/juja/microservices/keepers/service/KeepersService.java

```diff
@@ -64,6 +64,22 @@ public String addKeeper(KeeperRequest keeperRequest){
     }
 
     public Map<String, List<String>> getActiveKeepers() {
-        return keepersRepository.getActiveKeepers();
+        logger.debug("Service.getActiveKeepers after in, without any parameters.");
+        Map<String, List<String>> outMap = new HashMap<>();
+        List<Keeper> keepers = keepersRepository.getActiveKeepers();
+        for (Keeper keeper : keepers) {
+            List<String> directions = new ArrayList<>();
+            if(outMap.containsKey(keeper.getUuid())) {
+                directions = outMap.get(keeper.getUuid());
+                directions.add(keeper.getDirection());
+                outMap.replace(keeper.getUuid(),directions);
+            }else{
+                directions.add(keeper.getDirection());
+                outMap.put(keeper.getUuid(), directions);
+            }
+        }
+        logger.info("Get List<Keeper> : {}, and create Map<String, List<String>> : {} from income list.", keepers,outMap);
```

Comment:
please log the result of repository.getActiveKeepers() upper, right afte that method invocation. In this place (row 81) you can log only map.